### PR TITLE
Playbook section and Template Language Doc

### LINKED ISSUE #58: Bad Python Docs Link
44a6c55ba42203b08f9fa83cac2b4118117c886e fixed the link formatting, but https://docs.ab.bot/chatops/writing-skills/ still has a link to `xref:abbot-py-docs` instead of resolving to https://docs.ab.bot/reference/python/.

### PR DESCRIPTION
**Pull request by analogrelay**
This adds a Playbooks section to the docs and a brief overview doc describing the Template Language and what variables are available. It's not a full reference but it covers the basics.

In addition, I fixed a few docfx warnings and xref issues.
- Fixes #58

### CODE REVIEW

File: src/playbooks/template-language.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+Hey everyone, we got a new customer. They're in <#{{ trigger.outputs.channel.id }}>!
+```
+
+### Accessing the Customer that triggered the playbook
+
+If you trigger a playbook using the "New Customer Created" trigger, a `customer` output is available that contains information about the customer:
+
+| Variable | Description |
+| -------- | ----------- |
+| `trigger.outputs.customer.id` | The ID of the customer |
```
Comment (pmn) [inline src/playbooks/template-language.md:101]: Out of curiosity, is there ever a reason that an end user would need to know the ID of a customer?

Comment (pmn) [reply src/playbooks/template-language.md:101]: (I'm not saying it shouldn't be in the docs, just curious about its usefulness to end users)

**Comment by pmn** [comment]
This is a great first cut, thanks for getting it going

File: src/playbooks/template-language.md (hunk 1 above)
Comment (analogrelay) [reply src/playbooks/template-language.md:101]: Not really, no

Comment (analogrelay) [reply src/playbooks/template-language.md:101]: I'll remove it from the doc, it's really not useful information.

**Review by dahlbyk — APPROVED** [review]
General question on Playbook capitalization, but otherwise this looks correct to me so far.

**Comment by analogrelay** [comment]
> General question on Playbook capitalization, but otherwise this looks correct to me so far.

That's gonna be a constant thing for me 😅 . Maybe we need a linter.

**Pull request merged**